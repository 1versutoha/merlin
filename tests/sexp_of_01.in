["find", "use", "sexplib"]
["tell", "source", "type t = int with sexp_of\n"]
["tell", "end"]
["errors"]
["dump", "env"]
