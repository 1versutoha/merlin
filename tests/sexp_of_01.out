["return",true]
["return",null]
["return",{"line":2,"col":0}]
["return",[]]
["return",[["Line 1, characters 5-13","type t = int"],["Line 1, characters 18-25","val sexp_of_t : t -> Sexplib.Sexp.t"]]]
